bots/twitterbot.go

```diff
@@ -23,19 +33,36 @@ type TwitterBot struct {
 
 var _ Bot = (*TwitterBot)(nil)
 
+// A TwitterConfig contains the address of the named UNIX socket
+// through which the bot and the CONIKS server communicate,
+// the OAuth information needed to authenticate the bot with Twitter,
+// and the bot's reserved Twitter handle. These values are specified
+// in a configuration file, which is read at initialization time.
```

Comment:
Maybe make this clearer by adding ", which is in the TOML (Tom's Obvious, Minimal Language) format."?